### PR TITLE
git Issue #3930: Fix fieldset labels overflowing their container.

### LINKED ISSUE #3930: Seven: Fieldset labels overflow container when collapsed
**Description of the bug**
The theme settings headers do not display well as seen in the basis theme.

Basis_settings___Virtual_User_Group

**Steps To Reproduce**

1) On a fresh site, go to: /admin/appearance/settings/basis
2) Close the TABS AND BREADCRUMS section 
3) See how text overflows the container

**Expected behavior**

The container for the fieldset-title should expand to fit the the title when it drops to 2+ lines. 
Smaller text and/or not having the fieldset-title in all caps would make it fit better.

### PR DESCRIPTION
**Pull request by klonos**
Fixes https://github.com/backdrop/backdrop-issues/issues/3930

### CODE REVIEW

File: core/themes/seven/css/style.css [hunk 1]
```diff
@@ -645,33 +645,36 @@ tr.selected td {
...
-  padding-left: 0;
+fieldset:not(.collapsible) .fieldset-legend {
+  padding: .3em .3em .3em 1em; /* LTR */
+}
+[dir="rtl"] fieldset:not(.collapsible) .fieldset-legend {
+  padding: .3em 1em .3em .3em;
+}
+fieldset:not(.collapsible) .fieldset-legend,
+fieldset.collapsible:not(.collapsed) .fieldset-legend {
+  border-bottom: none;
```
Comment (klonos) [inline core/themes/seven/css/style.css:677]: This line removes the border between the fieldset legend and the fieldset content, to retain the current style:

screencapture-3930-lndo-site-444-admin-appearance-settings-basis-2020-05-19-01_27_46

If most people find this border acceptable/preferable, then we can remove this bit and have the border added.

**Comment by klonos** [comment]
With https://github.com/backdrop/backdrop-issues/issues/4318 merged, I'm refreshing the PR sandbox...

**Comment by klonos** [comment]
PR sandbox is borked 😞 ...

**Comment by quicksketch** [comment]
Merged into 1.x and 1.17.x. Thanks @BWPanda and @klonos!

**Pull request merged**